stix_shifter_modules/arcsight/configuration/config.json

```diff
@@ -0,0 +1,37 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "ArcSight"
+        },
+        "host": {
+            "type": "text"
+        },
+        "port": {
+            "type": "number",
+            "default": 443
+        },
+        "help": {
+            "type": "link",
+            "default": "<Help URL>"
+        },
+        "sni": {
+            "type": "text",
+            "optional": true
+        },
+        "selfSignedCert": {
+            "type": "password",
+            "optional": true
+        }
```

Comment:
'unmapped_fallback' attribute is now included in the connection object and is updated to the PR branch.

Reply:
We don't want to use the mapping fallback for the arc sight connector, so this should be removed from the arc sight config, that way it will use the global config which defaults the mapping fallback to false.

Reply:
@delliott90 @mdazam1942  The mapping fallback config param is removed and the change is updated to PR branch.